cli/src/main/java/com/devonfw/tools/ide/context/GitContextImpl.java

```diff
@@ -292,6 +292,17 @@ IdeContext getContext() {
 
     return this.context;
   }
+
+  /**
+   * Checks if there is a git installation and throws an exception if there is none
+   */
+  private void isGitInstalled() {
```

Comment:
Please rename to `verifyGitInstalled()` or `ensureGitIsInstalled()`.
According to Java Bean conventions this is a (boolean) getter but getters do not have `void` as return type.